backend/inspirehep/access_control.py

```diff
@@ -37,6 +37,17 @@ def can(self):
 
 class LiteraturePermissionCheck(SessionCatalogerPermission):
     def can(self):
-        if "Literature" not in self.record.get("_collections", []):
+        collections = {
```

Comment:
maybe make it clear what these collections are?
```suggestion
        non_private_collections = {
```